src/js/utils/StringUtil.js

```diff
@@ -66,6 +66,23 @@ const StringUtil = {
     return string.charAt(0).toUpperCase() + string.slice(1, string.length);
   },
 
+  humanizeArray: function (array, serialComma=true) {
```

Comment:
I'd say so, in both cases it's used to display arrays to the user. But maybe we just drop `arrayToJoinString` and use your implementation instead. Let's create a tech debt issue to track this and move on with this PR.